week0003/nhantd/python/week0003.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+Users=dict()
+def executeTransaction(Key, Name, Amount):
```

Comment:
New solution: I use class & functions to handle actions, which can be extended to deal with additional actions (through inheritance or adding new functions to object). Please review & let me know your feedback.